### PR TITLE
Add recipe for "sonata-project/form-extensions:1.0"

### PR DESCRIPTION
**Pull request by phansys**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by phansys** [comment]
Closing since the required bridges are forbidden by [this rule](https://github.com/symfony/recipes#creating-recipes):
>Note
>
>Don't create a recipe for Symfony bundles if the only configuration in the manifest is the registration of the bundle for all environments, as this is done automatically.

**Comment by phansys** [comment]
I can't add "sonata-project/doctrine-extensions" because there is no default configuration to add at "copy-from-recipe".
Even, I was not able to add "sonata-project/twig-extensions" because its type is "library" instead of "symfony-bundle":
>Package sonata-project/twig-extensions (1.0)
Type must be "symfony-bundle" as the manifest registers a bundle, "library" detected
Package sonata-project/twig-extensions (1.1.1)
Type must be "symfony-bundle" as the manifest registers a bundle, "library" detected

File: sonata-project/form-extensions/1.4/config/packages/sonata_form.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../1.0/config/packages/sonata_form.yaml
```
Comment (Nyholm) [inline sonata-project/form-extensions/1.4/config/packages/sonata_form.yaml:1]: Links are generally confusing. 

I dont mind if you duplicate the 2 lines of config here.

File: sonata-project/form-extensions/1.4/manifest.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Sonata\\Form\\Bridge\\Symfony\\SonataFormBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
```
Comment (Nyholm) [inline sonata-project/form-extensions/1.4/manifest.json:7]: Should we really copy everything in [this folder](https://github.com/sonata-project/twig-extensions/tree/1.x/src) to the user's application?

That seams wrong.

Comment (phansys) [reply sonata-project/form-extensions/1.4/manifest.json]: Thank you for the feedback!
It was just a bad copy / paste from other file: https://github.com/symfony/recipes-contrib/blob/956288e30be02cb7a4d70accb6e92059da26eff7/sonata-project/admin-bundle/3.31/manifest.json#L7
I've removed this line.

**Comment by phansys** [comment]
I am moving the PR to draft, since the configuration is not really needed (as it is currently holding only default values).
This recipe was added with the intention of resolving the bundle autodiscovery. In parallel, I provided a workaround to make the bundle discoverable: sonata-project/form-extensions#105.

**Comment by phansys** [comment]
They are not identical, there is a subtle difference at the bundle's QCN:
* 1.0 is using `Sonata\Form\Bridge\Symfony\Bundle\SonataFormBundle`
* 1.4 is using `Sonata\Form\Bridge\Symfony\SonataFormBundle`

**Comment by Nyholm** [comment]
Thank you. I missed that. 

 I’m happy to merge the recipe now.

**Comment by Nyholm** [comment]
@phansys are you done with the PR?

**Comment by phansys** [comment]
Yes @Nyholm. Thank you very much for your help :+1:

**Pull request merged**